docs/content/manual/pre-release/node/improve-node-failure-handling.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Improve Node Failure Handling By Automatically Force Delete Terminating Pods of StatefulSet/Deployment On Downed Node
+---
+1. Setup a cluster of 3 nodes
+1. Install Longhorn and set `Default Replica Count = 2` (because we will turn off one node)
+1. Create a SetfullSet with 2 pods using the command:
+    ```
+    kubectl create -f https://raw.githubusercontent.com/longhorn/longhorn/master/examples/statefulset.yaml
+    ```
+1. Create a volume + pv + pvc named `vol1` and create a deployment of default ubuntu named `shell` with the usage of pvc `vol1` mounted under `/mnt/vol1`
```

Comment:
Do we want to create a deployment(1 pod) with ```ubuntu``` image here?

Reply:
Thank you. Fixed